app/models/concerns/art_piece/to_csv.rb

```diff
@@ -0,0 +1,30 @@
+class ArtPiece
+  module ToCSV
+    extend ActiveSupport::Concern
+
+    included do
+      def self.to_csv
```

Comment:
Instead of a mixin, I'd recommend writing a class - ArtPieceCSVGenerator, and then calling it from self.to_csv within art_piece.rb